app/src/main/AndroidManifest.xml

```diff
@@ -78,6 +78,7 @@
       android:windowSoftInputMode="adjustResize" />
     <activity
       android:name=".app.profile.AdminPinActivity"
+      android:label="@string/admin_pin_activity_label"
```

Comment:
Rename `admin_pin_activity_label ` to `admin_pin_activity_title`

Reply:
@starboi02 Please reply to this comment

Reply:
@rt4914 thanks. I have made this change.